src/CLITests/OptionsUnitTests.cs

```diff
@@ -34,7 +35,7 @@ private int FailIfCalled(IEnumerable<Error> errors)
         private int ValidateOptions(Options options, string processName = null,
             int processId = 0, string outputDirectory = null, string scanId = null,
             string verbosity = null, bool showThirdPartyNotices = false,
-            int delayInSeconds = 0)
+            int delayInSeconds = 0, string customUia=null)
```

Comment:
spacing is a bit nonstandard
```suggestion
            int delayInSeconds = 0, string customUia = null)
```